pyproject.toml

```diff
@@ -9,6 +9,7 @@ requires = [
 [project]
 authors = [{ email = "allen.goodman@icloud.com", name = "Allen Goodman" }]
 dependencies = [
+    "numpy>=2.0.0",
```

Comment:
Let torch manage the numpy dependency

Reply:
I think torch doesn't depend on numpy now

Reply:
yeah that what I was seeing